test/test.js

```diff
@@ -32,53 +36,76 @@ function getCodeFromBundle(entries) {
 describe('rollup-plugin-multi-entry', () => {
   it('takes a single file as input', () =>
-    getCodeFromBundle('test/fixtures/0.js').then(code =>
-      includes(code, 'exports.zero = zero;')
-    ));
+    getModuleFromBundle('test/fixtures/named-0.js').then(moduleExports => {
+      deepStrictEqual(moduleExports, { zero: 0 });
+    }));
 
   it('takes an array of files as input', () =>
-    getCodeFromBundle(['test/fixtures/0.js', 'test/fixtures/1.js']).then(
-      code => {
-        includes(code, 'exports.zero = zero;');
-        includes(code, 'exports.one = one;');
-      }
-    ));
+    getModuleFromBundle([
+      'test/fixtures/named-0.js',
+      'test/fixtures/named-1.js'
+    ]).then(moduleExports => {
+      deepStrictEqual(moduleExports, { zero: 0, one: 1 });
+    }));
 
   it('allows an empty array as input', () =>
     getCodeFromBundle([]).then(code => doesNotInclude(code, 'exports')));
 
   it('takes a glob as input', () =>
-    getCodeFromBundle('test/fixtures/{0,1}.js').then(code => {
-      includes(code, 'exports.zero = zero;');
-      includes(code, 'exports.one = one;');
+    getModuleFromBundle('test/fixtures/named-{0,1}.js').then(moduleExports => {
+      deepStrictEqual(moduleExports, { zero: 0, one: 1 });
     }));
 
   it('takes an array of globs as input', () =>
-    getCodeFromBundle(['test/fixtures/{0,}.js', 'test/fixtures/{1,}.js']).then(
-      code => {
-        includes(code, 'exports.zero = zero;');
-        includes(code, 'exports.one = one;');
-      }
-    ));
+    getModuleFromBundle([
+      'test/fixtures/named-{0,}.js',
+      'test/fixtures/named-{1,}.js'
+    ]).then(moduleExports => {
+      deepStrictEqual(moduleExports, { zero: 0, one: 1 });
+    }));
 
   it('takes an {include,exclude} object as input', () =>
     getCodeFromBundle({
-      include: ['test/fixtures/*.js'],
-      exclude: ['test/fixtures/1.js']
+      include: ['test/fixtures/named-*.js'],
+      exclude: ['test/fixtures/named-1.js']
     }).then(code => {
       includes(code, 'exports.zero = zero;');
       doesNotInclude(code, 'exports.one = one;');
     }));
 
   it('allows to prevent exporting', () =>
     getCodeFromBundle({
-      include: ['test/fixtures/*.js'],
+      include: ['test/fixtures/named-*.js', 'test/fixtures/sideeffect.js'],
       exports: false
     }).then(code => {
       includes(code, `console.log('Hello, 2');`);
       doesNotInclude(code, 'zero');
       doesNotInclude(code, 'one');
     }));
+
+  it('allows exporting all default exports as an array', () =>
+    getModuleFromBundle({
+      include: ['test/fixtures/default-*.js'],
+      exports: 'array'
+    }).then(moduleExports => {
+      deepStrictEqual(moduleExports, [0, 1]);
+    }));
+
+  it('exports array entries in sorted order', () =>
+    getModuleFromBundle({
+      include: ['test/fixtures/default-1.js', 'test/fixtures/default-0.js'],
+      exports: 'array'
+    }).then(moduleExports => {
+      deepStrictEqual(moduleExports, [0, 1]);
```

Comment:
This doesn't feel right to me. I think what should happen is that _globs_ get expanded in place in sorted order, but everything else remains as it was. So in this case as there are no globs, each entry stays where it is in the list. What do you think?